### PR TITLE
Internal: Add Unit Test

### PR DESCRIPTION
**Pull request by 1yam**
Problem:
 - Some feature of Aleph-client doesn't have unit tests and https://github.com/aleph-im/aleph-client/issues/87

Solutions:
 - Add Unit test for aleph file upload
 - Add Unit test for aleph account path
 
 Current Coverage : 45 %

### CODE REVIEW

File: tests/unit/test_commands.py [hunk 1]
```diff
@@ -36,6 +37,13 @@ def test_account_export_private_key(account_file: Path):
     assert len(result.stdout.strip()) == 66
 
 
+def test_account_path():
```
Comment (hoh) [inline tests/unit/test_commands.py:40]: What is this testing ?

File: tests/unit/test_commands.py [hunk 2]
```diff
@@ -73,3 +81,19 @@ def test_message_find():
         b"bd79839bf96e595a06da5ac0b6ba51dea6f7e2591bb913deccded04d831d29f4"
         in result.stdout
     )
+
+
+def test_file_upload():
```
Comment (hoh) [inline tests/unit/test_commands.py:86]: Can you add a docstring that explains what this is testing ?

File: tests/unit/test_commands.py (hunk 1 above)
Comment (1yam) [reply tests/unit/test_commands.py:40]: This test check that path is set well

**Comment by MHHukiewitz** [comment]
Is superseded by #173

**Pull request closed without merging**